quax/examples/zero/_core.py

```diff
@@ -42,23 +42,24 @@ def materialise(self):
 
 @quax.register(lax.broadcast_in_dim_p)
 def _(
-    value: ArrayLike, *, broadcast_dimensions, shape
+    value: ArrayLike, *, broadcast_dimensions, shape, sharding=None
 ) -> Union[ArrayLike, quax.ArrayValue]:
     aval = jax.core.get_aval(value)
-    if isinstance(aval, jax.core.ConcreteArray) and aval.shape == () and aval.val == 0:
+    if False and aval.shape == () and aval.val == 0:
```

Comment:
Should this branch be removed?

Reply:
I must've left this in while debugging... let me see if I can revive the branch.

Reply:
Yes I think we should just remove it.